aspoet/src/main/kotlin/com/google/androidstudiopoet/generators/SourceModuleGenerator.kt

```diff
@@ -45,6 +46,26 @@ class SourceModuleGenerator(private val moduleBuildGradleGenerator: ModuleBuildG
         fileWriter.delete(projectBlueprint.projectRoot)
         fileWriter.mkdir(projectBlueprint.projectRoot)
 
+        if (projectBlueprint.generateBazelFiles) {
```

Comment:
Please extract this into `BazelWorkspaceBlueprint` and `BazelWorkspaceGenerator`, to keep this file small (although it should be even smaller :) ).